crates/core/tedge_actors/src/message_boxes.rs

```diff
@@ -184,25 +184,25 @@ impl<Input: Debug> LoggingReceiver<Input> {
 impl<Input: Send + Debug> MessageReceiver<Input> for LoggingReceiver<Input> {
     async fn try_recv(&mut self) -> Result<Option<Input>, RuntimeRequest> {
         let message = self.receiver.try_recv().await;
-        info!(target: &self.name, "recv {:?}", message);
+        trace!(target: &self.name, "recv {:?}", message);
```

Comment:
I would prefer to use the `DEBUG` level to trace the message received by an actor.

From my point of view:
- `TRACE` makes sense only if you are involved in the development. i.e. you are a developer.
- A user should be able to understand what's is going on with `DEBUG`.

Reply:
Changed inter-actor level communication log levels to `DEBUG`, leaving `TRACE` for clear cases for printing payloads where previously-printed `DEBUG` level logs leave enough context for the user to know what's happening.